data/pathfinder/paizo/roleplaying_game/ultimate_equipment/ue_equipmods.lst

```diff
@@ -98,40 +98,40 @@ Artisan's Tools (Weaponsmithing).MOD																													SOURCEPAGE:p.76
-Wyrmsbreath (Acid)							KEY:Special Ability ~ Wyrmsbreath / Acid ~ Shield								TYPE:Shield						COST:5000	VISIBLE:QUALIFY					SOURCEPAGE:p.123																			PRETYPE:2,Enhancement,Shield																																												SPROP:On command, the bearer can expend 1 to 5 charges from the shield to have it create a breath weapon in a 15 foot cone dealing 1d4 points of acid damage per charge expended (Reflex DC 11 half).
-Wyrmsbreath (Cold)							KEY:Special Ability ~ Wyrmsbreath / Cold ~ Shield								TYPE:Shield						COST:5000	VISIBLE:QUALIFY					SOURCEPAGE:p.123																			PRETYPE:2,Enhancement,Shield																																												SPROP:On command, the bearer can expend 1 to 5 charges from the shield to have it create a breath weapon in a 15 foot cone dealing 1d4 points of cold damage per charge expended (Reflex DC 11 half).
-Wyrmsbreath (Electricity)						KEY:Special Ability ~ Wyrmsbreath / Electricity ~ Shield							TYPE:Shield						COST:5000	VISIBLE:QUALIFY					SOURCEPAGE:p.123																			PRETYPE:2,Enhancement,Shield																																												SPROP:On command, the bearer can expend 1 to 5 charges from the shield to have it create a breath weapon in a 15 foot cone dealing 1d4 points of electricity damage per charge expended (Reflex DC 11 half).
-Wyrmsbreath (Fire)							KEY:Special Ability ~ Wyrmsbreath / Fire ~ Shield								TYPE:Shield						COST:5000	VISIBLE:QUALIFY					SOURCEPAGE:p.123																			PRETYPE:2,Enhancement,Shield																																												SPROP:On command, the bearer can expend 1 to 5 charges from the shield to have it create a breath weapon in a 15 foot cone dealing 1d4 points of fire damage per charge expended (Reflex DC 11 half).
+Adhesive									KEY:Special Ability ~ Adhesive ~ Armor										TYPE:Armor						COST:7000	VISIBLE:QUALIFY					SOURCEPAGE:p.114																			PRETYPE:2,ArmorEnhancement,Armor					!PRETYPE:1,EQMOD=Special Ability ~ Slick ~ Armor,EQMOD=Special Ability ~ Slick / Greater ~ Armor,EQMOD=Special Ability ~ Slick / Improved ~ Armor															SPROP:On command for 10 rds/day, climb speed of 20 feet and +2 enhancement bonus on combat maneuver checks made to disarm, grapple, reposition, steal, or trip when the wearer is using a natural weapon or unarmed strike.
+Balanced									KEY:Special Ability ~ Balanced ~ Armor										TYPE:Armor.Bracer			PLUS:1			VISIBLE:QUALIFY					SOURCEPAGE:p.115	PREMULT:2,[PRETYPE:1,ArmorEnhancement],[PREMULT:2,[PRETYPE:1,Armor],[PRETYPE:1,Light,Medium]],[PRETYPE:1,Bracer]																																																	SPROP:+4 to CMD vs overrun or trip and on Acrobatics check made to maintain balance. Dropping prone is a move action instead of a free action.
```

Comment:
This one falls in your exception - Enhancement needs to be left since Bracers are Misc. Actually, the way the logic looks Bracer would have never qualified...

Reply:
What did we decide on Bracers?  @Llisandur said bracers don't even require +1 to enhance with special qualities.   I suppose we need to break all the bracers out?

Reply:
Ruling seems to be that the only bracer that actually allows enhancing is the Bracers of Armor +1.   Changed the type on Bracers of Armor to ArmorEnhancement.